src/lib/Notification/TranslatableFlashBagNotificationHandler.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Notification;
+
+use Symfony\Component\Translation\TranslatorInterface;
+
+final class TranslatableFlashBagNotificationHandler implements TranslatableNotificationHandlerInterface
```

Comment:
Place where notifications  are stored it's implementation detail of `\EzSystems\EzPlatformAdminUi\Notification\NotificationHandlerInterface`:

```suggestion
final class TranslatableNotificationHandler implements TranslatableNotificationHandlerInterface
```